array.c

```diff
@@ -6764,23 +6773,22 @@ rb_ary_uniq_bang(VALUE ary)
 
 /*
  *  call-seq:
- *     ary.uniq                -> new_ary
- *     ary.uniq {|item| ...}   -> new_ary
- *
- *  Returns a new array by removing duplicate values in +self+.
- *
- *  If a block is given, it will use the return value of the block for comparison.
- *
- *  It compares values using their #hash and #eql? methods for efficiency.
- *
- *  +self+ is traversed in order, and the first occurrence is kept.
- *
- *     a = [ "a", "a", "b", "b", "c" ]
- *     a.uniq   # => ["a", "b", "c"]
- *
- *     b = [["student","sam"], ["student","george"], ["teacher","matz"]]
- *     b.uniq {|s| s.first}   # => [["student", "sam"], ["teacher", "matz"]]
- *
+ *    array.uniq -> new_array
+ *    array.uniq {|element| ... } -> new_array
+ *
+ *  Returns a new \Array containing those elements from +self+ that are not duplicates,
+ *  the first occurrence always being retained.
+ *
+ *  With no block given, identifies and omits duplicates using method <tt>eql?</tt>
+ *  to compare.
+ *    a = [0, 0, 1, 1, 2, 2]
+ *    a.uniq # => [0, 1, 2]
+ *
+ *  With a block given, calls the block for each element;
+ *  identifies and omits "duplicates",
```

Comment:
Probably want to use `duplicate values` instead of `"duplicates"`, to match the uniq! documentation.